app/services/submissions/search_service.rb

```diff
@@ -20,6 +20,16 @@ def call
 
   private
 
+    def sanitize_query(query)
```

Comment:
Nit: It feels to me like maybe this belongs in Search, not SearchService, as it's really to do with the internals of how `where_terms` is conducted and the interpolation there.

Reply:
Makes sense, just include here since this is where the bug was